jetty-util/src/main/java/org/eclipse/jetty/util/Index.java

```diff
@@ -19,18 +19,26 @@
 package org.eclipse.jetty.util;
 
 import java.nio.ByteBuffer;
+import java.util.Collections;
+import java.util.HashSet;
 import java.util.LinkedHashMap;
 import java.util.Map;
 import java.util.Set;
 import java.util.function.Function;
 import java.util.function.Supplier;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
 
 /**
  * An immutable String lookup data structure.
  * @param <V> the entry type
  */
 public interface Index<V>
 {
+    Set<Character> VISIBLE_ASCII_ALPHABET = Collections.unmodifiableSet(IntStream.range(0x20, 0x7f)
```

Comment:
I do not like this constant very much, I would prefer an extra helper method on the builder:

```java
 public Builder<V> withVisibleAsciiAlphabet()
 {
  alphabet = Collections.unmodifiableSet(...);
  return this;
 }
```